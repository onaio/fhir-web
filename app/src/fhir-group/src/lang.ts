import i18n, { namespace } from './mls';
import { Dictionary } from '@onaio/utils';

export type Lang = Dictionary<string>;

const lang: Lang = {};

/** recompute values */
function fill() {
  // Tooltips
  lang.TIP_REQUIRED_FIELD = i18n.t(`This is a required field`, { ns: namespace });

  // Errors
  lang.ERROR_OCCURRED = i18n.t(`An error occurred`, { ns: namespace });

  // Messages
  lang.MSG_ASSIGN_PRACTITIONERS = i18n.t(`Assigning Practitioners`, { ns: namespace });
  lang.MSG_ASSIGN_PRACTITONERS_SUCCESS = i18n.t(`Successfully Assigned Practitioners`, {
    ns: namespace,
  });
  lang.MSG_GROUP_UPDATE_SUCCESS = i18n.t(`Successfully Updated Group`, { ns: namespace });
  lang.MSG_GROUP_ADD_SUCCESS = i18n.t(`Successfully Added Group`, { ns: namespace });

  // Rendered text
  lang.ENTER_GROUP_NAME = i18n.t(`Enter a group name`, { ns: namespace });
  lang.ENTER_COMMENT = i18n.t(`Enter comment`, { ns: namespace });
  lang.ENTER_EXTRADETAILS = i18n.t(`Enter extra details`, { ns: namespace });
  lang.ENTER_ORGANIZATION = i18n.t(`Enter organization`, { ns: namespace });
  lang.LAST_UPDATED_DATE = i18n.t(`Last updated date`, { ns: namespace });
  lang.GROUP_NAME = i18n.t(`Group Name`, { ns: namespace });
  lang.STATUS = i18n.t(`Status`, { ns: namespace });
  lang.GROUP_MEMBERS = i18n.t(`Group Members`, { ns: namespace });
  lang.Active = i18n.t(`Active`, { ns: namespace });
  lang.ORGANIZATION = i18n.t(`Organization`, { ns: namespace });
  lang.NOORGANIZATION = i18n.t(`No organization`, { ns: namespace });
  lang.SAVING = i18n.t(`Saving`, { ns: namespace });
  lang.SAVE = i18n.t(`Save`, { ns: namespace });
  lang.ACTIVE = i18n.t(`Active`, { ns: namespace });
  lang.INACTIVE = i18n.t(`Inactive`, { ns: namespace });
  lang.SELECT_PRACTITIONER = i18n.t(`Select user (practitioners only)`, { ns: namespace });
  lang.CANCEL = i18n.t(`Cancel`, { ns: namespace });
  lang.EDIT_GROUP = i18n.t(`Edit Group`, { ns: namespace });
  lang.CREATE_GROUP = i18n.t(`Create Group`, { ns: namespace });
  lang.EDIT = i18n.t(`Edit`, { ns: namespace });
  lang.COMMENT = i18n.t(`Comment`, { ns: namespace });
  lang.NOCOMMENT = i18n.t(`No comment`, { ns: namespace });
  lang.EXTRADETAILS = i18n.t(`Extra Details`, { ns: namespace });
  lang.CREATE = i18n.t(`Create`, { ns: namespace });
  lang.IDENTIFIER = i18n.t(`Identifier`, { ns: namespace });
  lang.NO_GROUP_MEMBERS = i18n.t(`No group members`, { ns: namespace });
  lang.GROUPS = i18n.t(`Groups`, { ns: namespace });
  lang.SEARCH = i18n.t(`Search`, { ns: namespace });
  lang.MSG_GROUPS_UPDATE_SUCCESS = i18n.t(`Groups Update Successfully`, { ns: namespace });
  lang.MSG_GROUPS_ADD_SUCCESS = i18n.t(`Groups Added Successfully`, { ns: namespace });
}

// run it initial
fill();

// bind some events and fill values again (doing the magic you expect to happen magically)
i18n.on(`languageChanged`, () => {
  fill();
});

// export the const
export default lang;
