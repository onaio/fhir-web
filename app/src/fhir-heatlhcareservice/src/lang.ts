import i18n, { namespace } from './mls';
import { Dictionary } from '@onaio/utils';

export type Lang = Dictionary<string>;

const lang: Lang = {};

/** recompute values */
function fill() {
  // Tooltips
  lang.TIP_REQUIRED_FIELD = i18n.t(`This is a required field`, { ns: namespace });

  // Errors
  lang.ERROR_OCCURRED = i18n.t(`An error occurred`, { ns: namespace });

  // Messages
  lang.MSG_ASSIGN_PRACTITIONERS = i18n.t(`Assigning Practitioners`, { ns: namespace });
  lang.MSG_ASSIGN_PRACTITONERS_SUCCESS = i18n.t(`Successfully Assigned Practitioners`, {
    ns: namespace,
  });
  lang.MSG_HEALTHCARES_UPDATE_SUCCESS = i18n.t(`Successfully Updated Healthcares`, { ns: namespace });
  lang.MSG_HEALTHCARES_ADD_SUCCESS = i18n.t(`Successfully Added Healthcares`, { ns: namespace });

  // Rendered text
  lang.ENTER_HEALTHCARE_NAME = i18n.t(`Enter a healthcare name`, { ns: namespace });
  lang.ENTER_COMMENT = i18n.t(`Enter comment`, { ns: namespace });
  lang.ENTER_EXTRADETAILS = i18n.t(`Enter extra details`, { ns: namespace });
  lang.ENTER_ORGANIZATION = i18n.t(`Enter organization`, { ns: namespace });
  lang.LAST_UPDATED_DATE = i18n.t(`Last updated date`, { ns: namespace });
  lang.HEALTHCARE_NAME = i18n.t(`Healthcare Name`, { ns: namespace });
  lang.STATUS = i18n.t(`Status`, { ns: namespace });
  lang.HEALTHCARE_MEMBERS = i18n.t(`Healthcare Members`, { ns: namespace });
  lang.Active = i18n.t(`Active`, { ns: namespace });
  lang.ORGANIZATION = i18n.t(`Organization`, { ns: namespace });
  lang.NOORGANIZATION = i18n.t(`No organization`, { ns: namespace });
  lang.SAVING = i18n.t(`Saving`, { ns: namespace });
  lang.SAVE = i18n.t(`Save`, { ns: namespace });
  lang.ACTIVE = i18n.t(`Active`, { ns: namespace });
  lang.INACTIVE = i18n.t(`Inactive`, { ns: namespace });
  lang.SELECT_PRACTITIONER = i18n.t(`Select user (practitioners only)`, { ns: namespace });
  lang.CANCEL = i18n.t(`Cancel`, { ns: namespace });
  lang.EDIT_HEALTHCARE = i18n.t(`Edit Healthcare`, { ns: namespace });
  lang.CREATE_HEALTHCARE = i18n.t(`Create Healthcare`, { ns: namespace });
  lang.EDIT = i18n.t(`Edit`, { ns: namespace });
  lang.COMMENT = i18n.t(`Comment`, { ns: namespace });
  lang.NOCOMMENT = i18n.t(`No comment`, { ns: namespace });
  lang.EXTRADETAILS = i18n.t(`Extra Details`, { ns: namespace });
  lang.CREATE = i18n.t(`Create`, { ns: namespace });
  lang.IDENTIFIER = i18n.t(`Identifier`, { ns: namespace });
  lang.NO_HEALTHCARE_MEMBERS = i18n.t(`No healthcare members`, { ns: namespace });
  lang.HEALTHCARES = i18n.t(`Healthcares`, { ns: namespace });
  lang.SEARCH = i18n.t(`Search`, { ns: namespace });
}

// run it initial
fill();

// bind some events and fill values again (doing the magic you expect to happen magically)
i18n.on(`languageChanged`, () => {
  fill();
});

// export the const
export default lang;
