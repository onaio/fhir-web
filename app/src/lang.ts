import i18n from './mls';

export const ERROR_OCCURRED = i18n.t('An error occurred');
export const DOWNLOAD_CLIENT_DATA = i18n.t('Download Client Data');
export const CARD_SUPPORT = i18n.t('Card Support');
export const LOGIN = i18n.t('Login');
export const MANAGE_ACCOUNT = i18n.t('Manage account');
export const LOGIN_PROMPT = i18n.t('Please log in with one of the following providers');
export const USER_MANAGEMENT = i18n.t('User Management');
export const TEAMS = i18n.t('Teams');
export const LOCATIONS_UNIT = i18n.t('Locations unit');
export const LOCATIONS_UNIT_GROUP = i18n.t('Locations unit group');
export const PRODUCT_CATALOGUE = i18n.t('Product Catalogue');
export const MANIFEST_RELEASES = i18n.t('Manifest Releases');
export const DRAFT_FILES = i18n.t('Draft Files');
export const JSON_VALIDATORS = i18n.t('JSON Validators');
export const FORM_CONFIGURATION = i18n.t('Form Configuration');
export const PLANS = i18n.t('Plans');
export const ACTIVE = i18n.t('Active');
export const DRAFT = i18n.t('Draft');
export const COMPLETE = i18n.t('Complete');
export const TRASH = i18n.t('Trash');
export const LOCATIONS = i18n.t('Locations');
export const USERS = i18n.t('Users');
export const ADMIN = i18n.t('Admin');
export const MISSIONS = i18n.t('Missions');
export const WELCOME_TO_OPENSRP = i18n.t('Welcome to OpenSRP');
export const INVENTORY = i18n.t('Inventory');
export const SERVICE_POINT_INVENTORY = i18n.t('Service point inventory');
