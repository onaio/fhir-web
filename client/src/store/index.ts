// going to use this store initializer instead of the one in
// @opensrp/store package, for some reason its not working
// wehn passed to the redux provider
import { connectReducer, getConnectedStore } from '@onaio/connected-reducer-registry';
import { gateKeeperReducer, gateKeeperReducerName } from '@onaio/gatekeeper';
import reducerRegistry, { combine, Registry } from '@onaio/redux-reducer-registry';
import session, { reducerName as sessionReducer } from '@onaio/session-reducer';
import {
  reducerName as keycloakUsersReducerName,
  reducer as keycloakUsersReducer,
} from '@opensrp/store';
import { Reducer } from 'redux';
import { RouterState } from 'connected-react-router';

/** Initial reducers in the reducer registry */
const defaultReducers: Registry = {
  router: connectReducer as Reducer<RouterState>,
};

/** Add users reducer to registry */
defaultReducers[sessionReducer] = session;
defaultReducers[gateKeeperReducerName] = gateKeeperReducer;
// added this segment because registering from the user management
// package component is failing
defaultReducers[keycloakUsersReducerName] = keycloakUsersReducer;

/** The initial store for the reveal web app */
const store = getConnectedStore(defaultReducers);

/** Set listener to add reducers to store when registered */
reducerRegistry.setChangeListener((reducers) => {
  store.replaceReducer(combine(reducers));
});

export default store;
