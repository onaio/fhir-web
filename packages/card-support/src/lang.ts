import i18n, { namespace } from './mls';
import { Dictionary } from '@onaio/utils';

export type Lang = Dictionary<string>;
const lang: Lang = {};

/** recompute values */
function fill() {
  lang.ERROR_OCCURRED = i18n.t(`An error occurred`, { ns: namespace });
  lang.DOWNLOAD_CLIENT_DATA = i18n.t(`Download Client Data`, { ns: namespace });
  lang.CLIENT_LOCATION = i18n.t(`Client Location`, { ns: namespace });
  lang.ALL_LOCATIONS = i18n.t(`All Locations`, { ns: namespace });
  lang.CARD_STATUS = i18n.t(`Card Status`, { ns: namespace });
  lang.NEEDS_CARD = i18n.t(`Needs Card`, { ns: namespace });
  lang.CARD_NOT_NEEDED = i18n.t(`Card not needed`, { ns: namespace });
  lang.BOTH_NEEDS_CARD_CARD_NOT_NEEDED = i18n.t(`Both "Needs card" and "Card not needed"`, {
    ns: namespace,
  });
  lang.CARD_ORDER_DATE = i18n.t(`Card Order Date`, { ns: namespace });
  lang.CARD_ORDER_DATE_REQUIRED = i18n.t(`Please enter start date and end date`, { ns: namespace });
  lang.SELECT_CARD_ORDER_DATE = i18n.t(`Select Card Order Date`, { ns: namespace });
  lang.DOWNLOADING = i18n.t(`Downloading`, { ns: namespace });
  lang.DOWNLOAD_CSV = i18n.t(`Download CSV`, { ns: namespace });
  lang.NO_DATA_FOUND = i18n.t(`No data found`, { ns: namespace });
}

// run it initial
fill();

// bind some events and fill values again (doing the magic you expect to happen magically)
i18n.on(`languageChanged`, () => {
  fill();
});

// export the const
export default lang;
