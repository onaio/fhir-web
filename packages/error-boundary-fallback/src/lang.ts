import i18n, { namespace } from './mls';
import { Dictionary } from '@onaio/utils';

const lang: Dictionary<string> = {};

/** recompute values */
function fill() {
  lang.PAGE_TITLE = i18n.t(`An Error Occurred`, { ns: namespace });
  lang.PAGE_SUB_TITLE = i18n.t(
    `There has been an error. Itâ€™s been reported to the site administrators via email and should be fixed shortly. Thanks for your patience.'`,
    { ns: namespace }
  );
  lang.BUTTON_TITLE = i18n.t(`Back Home`, { ns: namespace });
}

// run it initial
fill();

// bind some events and fill values again (doing the magic you expect to happen magically)
i18n.on(`languageChanged`, () => {
  fill();
});

// export the const
export default lang;
