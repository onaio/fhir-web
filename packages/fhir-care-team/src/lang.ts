import i18n, { namespace } from './mls';
import { Dictionary } from '@onaio/utils';

export type Lang = Dictionary<string>;

const lang: Lang = {};

/** recompute values */
function fill() {
  // Tooltips
  lang.TIP_REQUIRED_FIELD = i18n.t(`This is a required field`, { ns: namespace });

  // Errors
  lang.ERROR_OCCURRED = i18n.t(`An error occurred`, { ns: namespace });

  // Messages
  lang.CARE_TEAMS_UPDATE_SUCCESS = i18n.t(`Successfully Updated Care Teams`, { ns: namespace });
  lang.CARE_TEAMS_ADD_SUCCESS = i18n.t(`Successfully Added Care Teams`, { ns: namespace });
  lang.CARE_TEAM_DELETE_SUCCESS = i18n.t(`Successfully Deleted Care Team`, { ns: namespace });
  lang.CONFIRM_DELETE = i18n.t(`Are you sure you want to delete this Care Team?`, {
    ns: namespace,
  });
  lang.YES = i18n.t(`Yes`, { ns: namespace });
  lang.NO = i18n.t(`No`, { ns: namespace });

  // Rendered text
  lang.STATUS = i18n.t(`Status`, { ns: namespace });
  lang.NAME = i18n.t(`Name`, { ns: namespace });
  lang.SAVING = i18n.t(`Saving`, { ns: namespace });
  lang.SAVE = i18n.t(`Save`, { ns: namespace });
  lang.ACTIVE = i18n.t(`Active`, { ns: namespace });
  lang.INACTIVE = i18n.t(`Inactive`, { ns: namespace });
  lang.CANCEL = i18n.t(`Cancel`, { ns: namespace });
  lang.EDIT_CARE_TEAM = i18n.t(`Edit Care Team`, { ns: namespace });
  lang.CREATE_CARE_TEAM = i18n.t(`Create Care Team`, { ns: namespace });
  lang.EDIT = i18n.t(`Edit`, { ns: namespace });
  lang.CREATE = i18n.t(`Create`, { ns: namespace });
  lang.IDENTIFIER = i18n.t(`Identifier`, { ns: namespace });
  lang.TEAMS = i18n.t(`Care Teams`, { ns: namespace });
  lang.CARE_TEAM_MEMBERS = i18n.t(`Care Teams`, { ns: namespace });
  lang.SEARCH = i18n.t(`Search`, { ns: namespace });
  lang.IDENTIFIER = i18n.t(`Identifier`, { ns: namespace });
  lang.CARE_TEAM_PAGE_HEADER = i18n.t(`FHIR Care Team`, { ns: namespace });
  lang.SUBJECT = i18n.t(`Subject`, { ns: namespace });
  lang.PARTICIPANTS = i18n.t(`Participant`, { ns: namespace });
}

// run it initial
fill();

// bind some events and fill values again (doing the magic you expect to happen magically)
i18n.on(`languageChanged`, () => {
  fill();
});

// export the const
export default lang;
