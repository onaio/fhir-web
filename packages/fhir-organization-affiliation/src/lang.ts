import i18n, { namespace } from './mls';
import { Dictionary } from '@onaio/utils';

export type Lang = Dictionary<string>;

const lang: Lang = {};

/** recompute values */
function fill() {
  // Tooltips
  lang.TIP_REQUIRED_FIELD = i18n.t(`This is a required field`, { ns: namespace });

  // Errors
  lang.ERROR_OCCURRED = i18n.t(`An error occurred`, { ns: namespace });

  // Messages
  lang.ORG_AFFILIATION_UPDATE_SUCCESS = i18n.t(`Successfully Updated Organization Affiliation`, {
    ns: namespace,
  });
  lang.ORG_AFFILIATION_ADD_SUCCESS = i18n.t(`Successfully Added Organization Affiliation`, {
    ns: namespace,
  });
  lang.ORG_AFFILIATION_DELETE_SUCCESS = i18n.t(`Successfully Deleted Organization Affiliation`, {
    ns: namespace,
  });
  lang.CONFIRM_DELETE = i18n.t(`Are you sure you want to delete this Organization Affiliation?`, {
    ns: namespace,
  });
  lang.YES = i18n.t(`Yes`, { ns: namespace });
  lang.NO = i18n.t(`No`, { ns: namespace });

  // Rendered text
  lang.STATUS = i18n.t(`Status`, { ns: namespace });
  lang.NAME = i18n.t(`Name`, { ns: namespace });
  lang.LOCATION_NAME = i18n.t(`Practitioner Name`, { ns: namespace });
  lang.ORGANIZATION_NAME = i18n.t(`Organization Name`, { ns: namespace });
  lang.SAVING = i18n.t(`Saving`, { ns: namespace });
  lang.SAVE = i18n.t(`Save`, { ns: namespace });
  lang.ACTIVE = i18n.t(`Active`, { ns: namespace });
  lang.INACTIVE = i18n.t(`Inactive`, { ns: namespace });
  lang.CANCEL = i18n.t(`Cancel`, { ns: namespace });
  lang.EDIT_ORG_AFFILIATION = i18n.t(`Edit Organization Affiliation`, { ns: namespace });
  lang.CREATE_ORG_AFFILIATION = i18n.t(`Create Organization Affiliation`, { ns: namespace });
  lang.EDIT = i18n.t(`Edit`, { ns: namespace });
  lang.CREATE = i18n.t(`Create`, { ns: namespace });
  lang.IDENTIFIER = i18n.t(`Identifier`, { ns: namespace });
  lang.ORGANIZATION_AFFILIATION = i18n.t(`Practitioner Roles`, { ns: namespace });
  lang.SEARCH = i18n.t(`Search`, { ns: namespace });
  lang.IDENTIFIER = i18n.t(`Identifier`, { ns: namespace });
  lang.ORG_AFFILIATION_PAGE_HEADER = i18n.t(`FHIR Organization Affiliation`, { ns: namespace });
  lang.ORGANIZATION = i18n.t(`Organization`, { ns: namespace });
  lang.LOCATION = i18n.t(`Practitioner`, { ns: namespace });
}

// run it initial
fill();

// bind some events and fill values again (doing the magic you expect to happen magically)
i18n.on(`languageChanged`, () => {
  fill();
});

// export the const
export default lang;
