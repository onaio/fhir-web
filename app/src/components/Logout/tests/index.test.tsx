import { mount, shallow } from 'enzyme';
import flushPromises from 'flush-promises';
import * as serverLogout from '@opensrp/server-logout';
import toJson from 'enzyme-to-json';
import React from 'react';
import { CustomLogout } from '..';
import { Provider } from 'react-redux';
import { MemoryRouter } from 'react-router';
import { store } from '@opensrp/store';
import * as notifications from '@opensrp/notifications';
import { act } from 'react-dom/test-utils';
import { ERROR_OCCURRED } from '../../../lang';

jest.mock('@opensrp/notifications', () => ({
  __esModule: true,
  ...Object.assign({}, jest.requireActual('@opensrp/notifications')),
}));

describe('components/Logout', () => {
  afterEach(() => {
    jest.clearAllMocks();
  });
  it('renders without crashing', () => {
    shallow(<CustomLogout />);
  });

  it('renders logout component', () => {
    const wrapper = mount(
      <Provider store={store}>
        <MemoryRouter>
          <CustomLogout />
        </MemoryRouter>
      </Provider>
    );
    expect(toJson(wrapper.find('CustomLogout'))).toMatchSnapshot();
  });

  it('calls logout with correct params', async () => {
    const mock = jest.spyOn(serverLogout, 'logout');
    mock.mockRejectedValue('Logout failed');
    const payload = {
      headers: {
        accept: 'application/json',
        authorization: 'Bearer null',
        'content-type': 'application/json;charset=UTF-8',
      },
      method: 'GET',
    };
    const logoutURL = 'https://opensrp-stage.smartregister.org/opensrp/logout.do';
    const keycloakURL =
      'https://keycloak-stage.smartregister.org/auth/realms/opensrp-web-stage/protocol/openid-connect/logout';
    mount(
      <Provider store={store}>
        <MemoryRouter>
          <CustomLogout />
        </MemoryRouter>
      </Provider>
    );
    await flushPromises();
    expect(mock).toHaveBeenCalledWith(payload, logoutURL, keycloakURL, 'http://localhost:3000');
  });

  it('handles logout failure correctly', async () => {
    jest.spyOn(serverLogout, 'logout').mockImplementation(() => Promise.reject('error'));
    const mockNotificationError = jest.spyOn(notifications, 'sendErrorNotification');
    mount(
      <Provider store={store}>
        <MemoryRouter>
          <CustomLogout />
        </MemoryRouter>
      </Provider>
    );
    await act(async () => {
      await flushPromises();
    });
    expect(mockNotificationError).toHaveBeenCalledWith(ERROR_OCCURRED);
  });
});
