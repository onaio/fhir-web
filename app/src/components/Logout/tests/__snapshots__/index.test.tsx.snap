// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Logout renders logout component 1`] = `
<CustomLogout>
  <Spin
    className="custom-spinner"
    size="large"
  >
    <Empty />
    <Spin
      className="custom-spinner"
      hashId="css-dev-only-do-not-override-k7429z"
      size="large"
      spinPrefixCls="ant-spin"
    >
      <div
        aria-busy={true}
        aria-live="polite"
        className="ant-spin ant-spin-lg ant-spin-spinning custom-spinner css-dev-only-do-not-override-k7429z"
      >
        <span
          className="ant-spin-dot ant-spin-dot-spin"
        >
          <i
            className="ant-spin-dot-item"
          />
          <i
            className="ant-spin-dot-item"
          />
          <i
            className="ant-spin-dot-item"
          />
          <i
            className="ant-spin-dot-item"
          />
        </span>
      </div>
    </Spin>
  </Spin>
</CustomLogout>
`;
